pages/people/[id]/records/referral/[[...stepId]].tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback, ReactElement } from 'react';
+import { useRouter } from 'next/router';
+
+import PersonView from 'components/PersonView/PersonView';
+import { useAuth } from 'components/UserContext/UserContext';
+import BackButton from 'components/Layout/BackButton/BackButton';
+import FormWizard from 'components/FormWizard/FormWizard';
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Seo from 'components/Layout/Seo/Seo';
+// import { addCase } from 'utils/api/cases';
+
+import formSteps from 'data/forms/asc-referral';
+
+import type { User, Resident } from 'types';
+
+type FormData = Resident & User & Record<string, unknown>;
+
+const Referral = (): ReactElement => {
+  const { query } = useRouter();
+  const { user } = useAuth() as { user: User };
+  const onFormSubmit = useCallback(
+    // (person) => async (formData: FormData) =>
```

Comment:
do we still need these comments?

Reply:
Confirm by Silvia that we're releasing it later. So we'll keep the function commented out for now.